main/k8s-cleanup.tf

```diff
@@ -0,0 +1,131 @@
+locals {
+  cleaner_name = "cleanup"
```

Comment:
cleaner -> cleanup? (you're right though, probably could have been "cleaner", but lets keep it consistent with everywhere else for now)

Reply:
there are other references to cleaner dotted around too, im sure your ide can sort it out :)

Reply:
yeah I'm not sure at what point I started typing cleaner vs cleanup 😅 - either way have changed everything to cleanup.